core/src/main/scala/org/elasticmq/ElasticMQError.scala

```diff
@@ -20,3 +20,8 @@ class MessageDoesNotExist(val queueName: String, messageId: MessageId) extends E
   val code = "MessageDoesNotExist"
   val message = s"Message does not exist: $messageId in queue: $queueName"
 }
+
+class ReceiptHandleIsInvalid(val queueName: String, receiptHandle: String) extends ElasticMQError {
```

Comment:
maybe simply: `InvalidReceiptHandle`?

Reply:
Fixed, although I left the string used in the api error response to be `ReceiptHandleIsInvalid`, so it conforms to https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_DeleteMessage.html